virusscanner/terraform/main.tf

```diff
@@ -0,0 +1,96 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "4.52.0"
```

Comment:
Can we change this to `version = ">= 4.0"`

Reply:
Sure can!

Reply:
Done